comms/AttributeImpl.h

```diff
@@ -19,7 +19,7 @@ class CAttributeBase : public clib::CLockableUnknown
     virtual const TCHAR *GetText(bool refresh, bool noBroadcast = false) const = 0;
     virtual const TCHAR *GetLabel() const = 0;
 
-    bool pluginFolderExists(const std::string & batchFile, boost::filesystem::path & foundFolder, int level, bool pluginFolder) const
+    bool pluginFolderExists(const std::string & batchFile, boost::filesystem::path & foundFolder, int level, bool typeFolder, const std::string & subFolder, const std::string & subSubFolder) const
```

Comment:
It  seems we could have done this with a single relative subfolder argument. 
I'm not sure why we had to split out the sub and the subsubfolder, but why not accept a single subFolder argument and tokenize based on the local system folder delimiter (if indeed we do need to split the path)?